packages/@lwc/template-compiler/src/parser/index.ts

```diff
@@ -749,6 +749,10 @@ export default function parse(source: string, state: State): TemplateParseResult
                 }
             }
 
+            // the if branch handles
+            // 1. All attributes for standard elements except 1 case are handled as attributes
+            //    * value and checked attribute for input element is treated as property
```

Comment:
> * value and checked attribute for input element is treated as property

This is an internal detail of this function. I wouldn't add a comment here if we change the implementation.

```suggestion
```